pkg/features/kube_features.go

```diff
@@ -680,7 +674,6 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	CSINodeInfo:                    {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.19
 	BlockVolume:                    {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.20
 	StorageObjectInUseProtection:   {Default: true, PreRelease: featuregate.GA},
-	SupportIPVSProxyMode:           {Default: true, PreRelease: featuregate.GA},
```

Comment:
Based on https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecation I think we need to set `LockToDefault` for 2 releases before we can remove it.

Reply:
Deprecation talks about announcing the deprecation (which we did in CHANGELOG/CHANGELOG-1.10.md) and about a warning when used (which happens by nature of being marked GA).

Deprecation also says "must be documented in both in the release notes and the corresponding CLI help" but CLI help filters out GA and deprecated gates.

Given the error is in the gates lib, I think we should be OK to delete this, and then we can fix the doc and the lib.

Disaagree?

Reply:
I think in this case I agree with that, as there are no further references in other parts of the code about the Feature gate.

But in case of https://github.com/kubernetes/kubernetes/pull/94581 I actually don't know if we should remove (as the Feature Gate went to GA in v1.18) or only mark it as LockToDefault: true, as that PR also removes some logic from the code. This also wasn't announced in the CHANGELOG/CHANGELOG-1.18.md when it turned into GA, should it be?

Reply:
> a warning when used (which happens by nature of being marked GA).

TIL :) 

In rule #7 of the Deprecation Policy:

> GA: the feature gate is deprecated (see "Deprecation") and becomes non-operational.

Does "non-operational" mean we don't check the gate anywhere or does it also imply the gate is locked? If the former, then sgtm to remove it now.